cmd/rm-main.go

```diff
@@ -307,7 +316,8 @@ func mainRm(ctx *cli.Context) error {
 	isRecursive := ctx.Bool("recursive")
 	isFake := ctx.Bool("fake")
 	isStdin := ctx.Bool("stdin")
-	older := ctx.Int("older-than")
+	olderThan := ctx.Int("older-than")
+	newerThan := ctx.Int("newer-than")
```

Comment:
Is it allowed to use `older-than` and `newer-than` flags together? If true, we would need to update help documents with examples, else we need to error out properly here.  This comment is applicable for all mc commands here.

Reply:
> Is it allowed to use older-than and newer-than flags together?

Yeah, it is allowed, for example if a user wants to avoid selecting objects belonging to a known time frame, like avoid mirroring all objects wrongly updated in the whole last week.

> If true, we would need to update help documents with examples,

I am the one who don't prefer adding too much number of examples, I already added for --older-than & --newer-than.

Reply:
I see `older-than` and `-newer-than` as separate examples.  Could you point me where both are in used?

Reply:
sorry, I mean I already add two separate examples for --older-than and --newer-than and I just think we don't need to add another one to keep the number of examples reduced

Reply:
IMO at least one example needs to be added to avoid question from users about using both options.

However please update https://github.com/minio/mc/blob/master/docs/minio-client-complete-guide.md for these options.

Reply:
> However please update https://github.com/minio/mc/blob/master/docs/minio-client-complete-guide.md for these options.

oups yeah, we didn't update that doc for a while now, I see at least mb, mirror, stat commands not updated, let's do that in another PR to review and fix everything at one, I'll open a new issue for that, what do you think ?

Reply:
OK